adds CAPA templates

laurencejackson opened the pull request:
ADds the CAPA templates to the project template

including both the CAPA.yml template (only available on public repos) and the CAPA_markdown.md template as a backup for use on private repos

Resolves #28

laurencejackson commented:
Thanks @AnilMistry, this PR adds both the CAPA form (.yml) and the CAPA markdown template. On public repositories, both should be available, and on private repositories, only the markdown should be available. 

Both templates offer the same information, but I think the yml form is a bit nicer to fill in when available. does it make sense to offer both in the project_template and devs can decide whether to remove one or leave both? Or are we moving to just use the markdown version since its available to all repos?

AnilMistry commented:
OK that makes sense. 
I think the markdown needs updating to include timeframes section at the bottom like in:
https://github.com/GSTT-CSC/CSC-QMS/blob/main/.github/ISSUE_TEMPLATE/CAPA_md.md

Although its slightly nice to fill out, the yml template removes the ability to create a root cause analysis 5-Whys tables, and stops the ability to upload any documents to the issue, which may end up being useful down the line. I'm also unsure of whether the markdown fields allow pictures to be put in.

laurencejackson commented:
I think the yml form format can support all the features we need it to but i think you're right and the best thing to do is probably the simplest - and only support the md format for now. If in the future github rolls the yml format out to private repos then we can revisit this. 

So for now I have
- removed the yml form
- updated the md template to match the QMS repo version
